### ISSUE TITLE
DetailsList - NVDA non interactive column header pronounce as clickable

### ISSUE THREAD

**Issue by MdAmeerMind** [OfficeDev/office-ui-fabric-react#11828]
**Actual behavior:**
NVDA is pronouncing column header name with word clickable

**Expected:**
If column is non interactive (No Column sorting is there), then NVDA should read it only column name. 

Code repro
https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/compact

In Above example, Name and value column are non intractable, but NVDA is pronounce it as clickable.

**Comment by khmakoto** [comment]
Hi @MdAmeerMind, thanks for filing this issue! I can confirm that this is happening but I can't seem to repro it with any of our supported matrices of browsers+screen readers. Both Narrator+Old Egde and Narrator+New Edge don't pronounce this as clickable. Even using NVDA with these browsers works just fine. Because of that I'd recommend filing an issue with NVDA/Chrome since it seems that we're doing everything fine on our side.

**Comment by feerrenrut** [comment]
I've tested this with both NVDA+Firefox and NVDA+Chrome. The dev tools for both show click handlers registered for this column header. It seems to me the error is in the sample, a click handler is present that is not useful. If you remove the click handlers from this node in Chrome, clickable is no longer reported.  What would you suggest a screen reader do when a node is clickable?

To reproduce with NVDA + Chrome:
1. Open the sample in chrome
1. press `NVDA+space` to to ensure you are in browse mode (you'll hear a beep, or "browse mode" depending on your settings). 
1. Press `T` to jump to the table
1. Press down arrow to move through the column headers. You will hear "clickable" reported.

Open Chrome dev tools and focus the element, select the "event listeners tab" you'll see various click handlers. 
Remove them all and repeat steps above. Clickable is not reported. 
Note, as far as I can tell the click handlers are just stubs as an example. Perhaps a better example would have them do something?

Here are some screenshots to illustrate:

### Chrome:

To save myself time I just removed all these click handlers.
![image](https://user-images.githubusercontent.com/3491346/73654428-f733ff80-468b-11ea-944b-3933b7bafe6b.png)


### Firefox: 
An empty handler. 
![image](https://user-images.githubusercontent.com/3491346/73654555-53971f00-468c-11ea-99c4-07f3d5ec2a56.png)

Accessibility tool showing click action.
![image](https://user-images.githubusercontent.com/3491346/73654637-85a88100-468c-11ea-9ce2-b284fa3e15ea.png)

**Comment by khmakoto** [comment]
Hi @feerrenrut, can you show me what the contents of that `click` action look like? That would help me get a better idea of of where this info is coming from.

**Comment by feerrenrut** [comment]
@khmakoto The best I can do is what the screenshots provide. You can use Firefox / Chrome to investigate further. These click handlers are in the sample, and the javascript is obfuscated. You don't need a screen reader running to see that there are click handlers. Please ask the creators of the sample to explain it. 

Here is a screenshot of the click handlers with Edge:
![image](https://user-images.githubusercontent.com/3491346/73832562-b31a3980-4808-11ea-8fbb-c1c15a98d92c.png)

Sorry to say this, but I can't investigate this issue for you. Microsoft has ample resources to understand this issue. If you can show that NVDA could be doing something better in this case, I'll be happy to spend more time on it. My opinion is that these click handlers are unnecessary and should be removed, or a suitable action should be taken to demonstrate that they are useful in this sample.

**Comment by khmakoto** [comment]
Hi @feerrenrut, I understand, thanks for the screenshots. I'll continue looking into this in the coming days.

**Comment by cschlechty** [comment]
Given there's a click event on the header, the AT behavior is expected. The click will move focus for the focus zone so if you tab it'll focus that cell in the row, which seems by design.
